src/SFA.DAS.Charities.Api/Properties/launchSettings.json

```diff
@@ -0,0 +1,27 @@
+﻿{
+  "iisSettings": {
+    "windowsAuthentication": false, 
+    "anonymousAuthentication": true, 
+    "iisExpress": {
+      "applicationUrl": "http://localhost:13855",
```

Comment:
Incorrect port. Should match up with the Kestrel port

Reply:
fixed